### PR TITLE
Remove comment about working directory in Dockerfile

### PR DESCRIPTION
**Pull request by jmleroux**
## Description

The previous comment was not correct. Plus, the command itself is explicit, so the best soltuion is to not have any comment.

## Definition Of Done

| Q                                 | A
| --------------------------------- | ---
| Added tests                       | -
| Changelog updated                 | -
| Documentation                     | Yes
| Fixed tickets                     | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: php/7.1/Dockerfile [hunk 1]
```diff
@@ -46,7 +46,7 @@ RUN chmod +x /usr/local/bin/composer
 COPY ./files/entrypoint.sh /usr/local/bin
 RUN chmod a+x /usr/local/bin/entrypoint.sh
 
-# Define "docker" as current user
+# Change working directory to use the default user (ie 'docker') home
```
Comment (anaelChardan) [inline php/7.1/Dockerfile]: It is not really the default user, we add one, but the config may differs as it is configured outside of the dockerfile

Comment (damien-carcel) [reply php/7.1/Dockerfile]: Thanks for the fix @jmleroux . Anael is right,  maybe just `to use the home of "docker" user` is enough.
Could you also make the change on the other PHP versions of the `akeneo/php` image, please?

Comment (docteurklein) [reply php/7.1/Dockerfile]: maybe it's even better to remove this comment

Comment (jmleroux) [reply php/7.1/Dockerfile]: Totally agree @docteurklein . The command is explicit on its own. Another example of the maintenance debt of a comment :smile:

**Comment by jmleroux** [comment]
I rebased my branch and edited the PR description

**Comment by damien-carcel** [comment]
I am gonna be "lourd", but you forgot the new PHP 7.2 Dockerfile :smile:.
After that it will be GTM.

**Review by damien-carcel — APPROVED** [review]

**Comment by damien-carcel** [comment]
Merged. Thank you for this fix @jmleroux

**Pull request merged**